lms/djangoapps/ora_staff_grader/mock/data/ora_metadata.json

```
@@ -25,6 +25,13 @@
                         },
                         {
                             "orderNum": 1,
+                            "name": "good",
+                            "label": "good",
```

The name and label often have different capitalization. This is also helpful to make sure we're accessing the right field, unlike when they're both the same.

```suggestion
                            "name": "good",
                            "label": "Good",
```